modules/rosa-sdpolicy-am-cluster-self-service.adoc

```diff
@@ -16,4 +16,4 @@ Customers can self-service their clusters, including, but not limited to:
 * Add or remove machine pools and configure autoscaling
 * Define upgrade policies
 
-These tasks can be self-serviced using the `rosa` CLI utility.
\ No newline at end of file
+You can perform these self-service tasks using the {product-title} (ROSA) CLI, `rosa`, utility.
```

Comment:
Similar issue to separating "tool" from "CLI" as above - probably want to keep it "CLI utility, `rosa`" if you need to keep "utility".